src/website/views/components/alert/_examples.pug

```diff
@@ -233,6 +233,132 @@ p.-text
         </div>
       </div>
 
+h4 Spinner
+p.-text
+  | Use the spinner attribute to let users know when information is saving or loading.
+.example.-mb--3
+  .-p--1
+    chi-alert(icon='circle-info', spinner).-m--2
+      | This is a base alert
+    each val, type in {success:'circle-check', danger:'circle-x', warning:'warning', info:'circle-info', muted:'circle-info'}
+      chi-alert.-m--2(color=type, icon=val, spinner)
```

Comment:
@wojciechwlodarczyklumen please remove all the icons in the spinners wc examples as they are not needed. Also check the tests